piksi_tools/console/update_view.py

```diff
@@ -646,6 +651,15 @@ def _compare_versions(self):
 
         fw_update_prompt.run()
 
+    # Check if firmware successfully upgraded and notify user if so.
+    if not self.fw_outdated and self.last_call_fw_version is not None and \
+        self.last_call_fw_version != local_stm_version:
+      print "Firmware successfully upgraded from %s to %s." %
```

Comment:
Can you make a write_call to the text window on the update tab for this message as well?